AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -24,37 +25,56 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
     private let mutationEventPublisher: MutationEventPublisher
     private let outgoingMutationQueue: OutgoingMutationQueueBehavior
 
+    private var reconciliationQueueCancellable: AnyCancellable?
+
+    private let remoteSyncTopicPublisher: PassthroughSubject<RemoteSyncEngineEvent, DataStoreError>
+    var publisher: AnyPublisher<RemoteSyncEngineEvent, DataStoreError> {
+        return remoteSyncTopicPublisher.eraseToAnyPublisher()
+    }
+
     /// Synchronizes startup operations
     let syncQueue: OperationQueue
 
     // Assigned at `setUpCloudSubscriptions`
     var reconciliationQueue: IncomingEventReconciliationQueue?
+    var reconciliationQueueFactory: IncomingEventReconciliationQueueFactory
 
     /// Initializes the CloudSyncEngine with the specified storageAdapter as the provider for persistence of
```

Comment:
Update the comment.

Reply:
Hmm..  `reconiliationQueue` is still assigned in setupCloudSubscriptions.  Hmm not exactly sure which comment you are referring to?

Reply:
Nvm, I saw the new parameters added. Thought we should mention that in the comments.